favorite-patterns/favorite-patterns.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * お気に入りのパターンを呼び出して登録
+ */
+
+/**
+ * API からデータを読み込み
+ */
+function vbp_get_pattern_api_data() {
+    $options    = get_option( 'vk_block_patterns_options' );
+    $user_email = $options['patternLibraryUserName'];
```

Comment:
```suggestion
    $user_email = $options['patternLibraryUserEmail'];
```
@drill-lancer Email 保存なら保存フィールドは Emailで